### PR TITLE
Updated `LeftEventArgs` documentation.

### PR DESCRIPTION
**Pull request by ALEXWARELLC**
I have updated the documentation for 'LeftEventArgs'.

Hopefully this can close issue #1856.

### CODE REVIEW

File: Exiled.Events/EventArgs/Player/LeftEventArgs.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Exiled.Events.EventArgs.Player
     using API.Features;
 
     /// <summary>
-    /// Contains all information after a <see cref="Player"/> disconnects from the server.
+    /// Contains all information after a <see cref="Player"/> disconnects from the server. This class is inherited from JoinedEventArgs.
```
Comment (NaoUnderscore) [inline Exiled.Events/EventArgs/Player/LeftEventArgs.cs:13]: Use `<br>` or `<para>`.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/Player/LeftEventArgs.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace Exiled.Events.EventArgs.Player
...
     /// </summary>
     public class LeftEventArgs : JoinedEventArgs
```
Comment (louis1706) [inline Exiled.Events/EventArgs/Player/LeftEventArgs.cs:15]: Add TODO: Remove inherited from JoinedEventArgs
instead of this documentation change

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/Player/LeftEventArgs.cs (hunk 2 above)
Comment (ALEXWARELLC) [reply Exiled.Events/EventArgs/Player/LeftEventArgs.cs:15]: Expand on this?

File: Exiled.Events/EventArgs/Player/LeftEventArgs.cs (hunk 1 above)
Comment (ALEXWARELLC) [reply Exiled.Events/EventArgs/Player/LeftEventArgs.cs]: I have used `<br>`

File: Exiled.Events/EventArgs/Player/LeftEventArgs.cs (hunk 2 above)
Comment (ALEXWARELLC) [reply Exiled.Events/EventArgs/Player/LeftEventArgs.cs:17]: Never mind, I got it in there.

**Review by NotIntense — CHANGES_REQUESTED** [review]
"This class is inherited from JoinedEventArgs" is not needed

**Comment by NotIntense** [comment]
https://github.com/Exiled-Team/EXILED/pull/2062

**Pull request closed without merging**